Realm/RealmSwift/Migration.swift

```diff
@@ -54,23 +54,26 @@ public class Migration {
 
     private var rlmMigration: RLMMigration
 
-    init(_ rlmMigration: RLMMigration) {
+    internal init(_ rlmMigration: RLMMigration) {
         self.rlmMigration = rlmMigration
     }
 }
 
 public class MigrationObject : Object {
-    subscript(key: String) -> AnyObject? {
-        if (self.objectSchema[key].type == RLMPropertyType.Array) {
-            return listProperties[key]
+    public override subscript(key: String) -> AnyObject? {
```

Comment:
I ran the tests, but I assume those aren't quite ready yet for the new Migration stuff?